utils_test.py

```diff
@@ -198,18 +215,20 @@ def test_capitalize_string(self):
             ['Partially', 'Partially'],
             ['miDdle', 'MiDdle'],
             ['2be', '2be'],
-        ]
+        ] # type: List[Any]
```

Comment:
```suggestion
        ] # type: List[Union[None,Text]]
```

Reply:
It should be `List[List[Union[None, Text]]]`. Added it.